x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/DownsampleMetrics.java

```diff
@@ -40,6 +42,11 @@ public DownsampleMetrics(MeterRegistry meterRegistry) {
         this.meterRegistry = meterRegistry;
     }
 
+    @Inject
+    public DownsampleMetrics(TelemetryProvider telemetryProvider) {
```

Comment:
It appears the other constructor isn't needed, so this could be the lone ctor and not need `@Inject`?